Description: Set LANGUAGE when running diff tests.
Origin: commit, revision id: jelmer@jelmer.uk-20171005225153-m0yy2y26pr7hsasl
Author: Jelmer VernooÄ³ <jelmer@jelmer.uk>
Last-Update: 2017-10-05
Applied-Upstream: no
X-Bzr-Revision-Id: jelmer@jelmer.uk-20171005225153-m0yy2y26pr7hsasl

=== modified file 'breezy/tests/test_diff.py'
--- old/breezy/tests/test_diff.py	2017-06-11 02:27:01 +0000
+++ new/breezy/tests/test_diff.py	2017-10-05 22:51:53 +0000
@@ -326,8 +326,9 @@
         cmd = ['diff', '-u', '--binary', 'old', 'new']
         with open('old', 'wb') as f: f.write('\x00foobar\n')
         with open('new', 'wb') as f: f.write('foo\x00bar\n')
-        pipe = subprocess.Popen(cmd, stdout=subprocess.PIPE,
-                                     stdin=subprocess.PIPE)
+        pipe = subprocess.Popen(
+                cmd, stdout=subprocess.PIPE,
+                stdin=subprocess.PIPE, env={'LANGUAGE': 'C.UTF-8'})
         out, err = pipe.communicate()
         # We should output whatever diff tells us, plus a trailing newline
         self.assertEqual(out.splitlines(True) + ['\n'], lines)

